docs/src/calling-contracts/cost-estimation.md

```diff
@@ -26,3 +26,4 @@ Below are examples that show how to get the estimated transaction cost from sing
 
 The transaction cost estimation can be used to set the gas limit for an actual call, or to show the user the estimated cost.
 
+> **Note:** whenever you perform an action that starts a transaction (contract deployment, contract call, asset transfer), the SDK will automatically estimate the fee behind the scenes and prepare the transaction accordingly. A side-effect of this is that transactions require the wallet to own at least an amount of 1 of the base asset, even if the gas cost is set to 0 via `TxParameters`. 
```

Comment:
> require the wallet to own at least an amount of 1 of the base asset

Just double checking; this means at least one coin but not necessarily with its _value_ being 1? e.g. if the base asset is ETH, you'd need at least 1 ETH "coin" but the value could be something reasonable, like `0.001`?

Reply:
IIUC `amount` is an integer so the smallest I can assume here is 1. I don't know how that will convert to ETH or some other asset, but the smallest we can represent is always an amount of 1.

Reply:
Yeah, right. I guess we can worry about this later then!